### PR TITLE
change routing link (Fixed 4761)

### LINKED ISSUE #4761: Enterprise delete from detail page refirects to team list page
- create an enterprise.
- click on that enterprise.
- click on delete within enterprise detail page
Screenshot from 2019-09-05 20-56-28

- it will redirect to the team list page
- it should redirect to the enterprise list page.

### PR DESCRIPTION
**Pull request by wanglihe2010**
fixed #4761

### CODE REVIEW

**Comment by wanglihe2010** [comment]
I just noticed I created current branch from another issue branch with unmerged  commit. What is the best way to fix it? Should I discard the branch and create a new one?

File: src/app/teams/teams.component.ts [hunk 1]
```diff
@@ -47,6 +46,8 @@ export class TeamsComponent implements OnInit, AfterViewInit {
   deleteDialog: any;
   readonly myTeamsFilter = this.route.snapshot.data.myTeams ? 'on' : 'off';
   mode: 'team' | 'enterprise' = this.route.snapshot.data.mode || 'team';
+  displayedColumns = this.planetType === 'community' && this.mode === 'enterprise' ?
```
Comment (paulbert) [inline src/app/teams/teams.component.ts]: This is a change from #4776.  Please make sure that when you create a new branch you are on the master branch (`git checkout master`)

File: src/app/teams/teams.component.html [hunk 2]
```diff
@@ -39,7 +39,7 @@ <h3>
           {{element.doc.createdDate | date: 'medium'}}
         </mat-cell>
       </ng-container>
-      <ng-container matColumnDef="teamType">
+      <ng-container matColumnDef="teamType" *ngIf="!(planetType === 'community' && mode === 'enterprise')">
```
Comment (paulbert) [inline src/app/teams/teams.component.html]: Same, this is from #4776

File: src/app/teams/teams-view.component.ts [hunk 3]
```diff
@@ -192,7 +192,11 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
             this.planetMessageService.showMessage(`You have ${config.successMsg} ${displayName}`);
             this.team = change === 'course' ? res : this.team;
             if (res.status === 'archived') {
-              this.router.navigate([ '/teams' ]);
+              if (this.mode === 'team') {
```
Comment (paulbert) [inline src/app/teams/teams-view.component.ts]: There is a `goBack` method which takes care of this logic.  Please use that so it makes it more maintainable (one place to change if we need to add logic for the routing).

**Comment by lmmrssa** [comment]
> I just noticed I created current branch from another issue branch with unmerged commit. What is the best way to fix it? Should I discard the branch and create a new one?

You should always checkout to master, update master and create new branch when you start working on any issue. You can do rebasing of this branch if you are familiar with it. Otherwise easiest is to create new branch and close this one after you put changes to new branch

**Comment by paulbert** [comment]
Closing in favor of #4807

**Pull request closed without merging**